ext/opcache/ZendAccelerator.h

```diff
@@ -55,6 +55,14 @@
 #include "zend_accelerator_hash.h"
 #include "zend_accelerator_debug.h"
 
+#ifdef HAVE_STDATOMIC_H
+#ifdef __cplusplus
```

Comment:
Nit: Indentation would be nice, as I find it easier to read.

Reply:
Changed.